### PR TITLE
[CARBONDATA-549] code improvements for bigint, decimal value compression

### PR DESCRIPTION
**Pull request by JihongMA**
misc code changes for bigint, decimal value compression

mvn clean verify -- successful

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by JihongMA** [comment]
ok to test

**Comment by JihongMA** [comment]
add to whitelist

**Comment by chenliang613** [comment]
add to whitelist

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/283/

File: core/src/main/java/org/apache/carbondata/core/datastorage/store/compression/WriterCompressModel.java [hunk 1]
```diff
@@ -88,15 +88,15 @@ public void setCompType(ValueCompressionUtil.COMPRESSION_TYPE[] compType) {
   /**
    * @return the changedDataType
```
Comment (ravipesala) [inline core/src/main/java/org/apache/carbondata/core/datastorage/store/compression/WriterCompressModel.java]: Please change the comment as well

**Comment by ravipesala** [comment]
LGTM except small comment,  Thanks for refactoring the code.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/292/

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**